src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -226,7 +226,9 @@ public function canBeEnabled()
             ->beforeNormalization()
                 ->ifArray()
                 ->then(function ($v) {
-                    $v['enabled'] = isset($v['enabled']) ? $v['enabled'] : true;
+                    if (!isset($v['enabled'])) {
+                        $v['enabled'] = empty($v) ? false : true;
```

Comment:
Can also be written as `$v['enabled'] = !empty($v);`

Reply:
damn, you're right :D